### DISCUSSION TITLE
Question About the Showcase Menu Entry

### DISCUSSION THREAD

**Discussion by ghost** [colinwilson/lotusdocs#27] [Q&A] [answered]
I'm beginning to populate content using this theme as a module.

In this [`hugo.toml`][1], I noticed another menu entry for "[showcase][2]". Is this going to be an independent section like `docs/`, i.e., only pages from `content/showcase/` will be in the left menu entry, or will it be a part of `content/`, i.e., navigating to either `content/docs/` or `content/showcase/` will show a combined menu entry for both sections ?

This would give me an idea of organizing my content.

[1]: https://github.com/colinwilson/lotusdocs/blob/release/exampleSite/hugo.toml
[2]: https://github.com/colinwilson/lotusdocs/blob/49afc59b7165da96e3f4e9ca8713107c311427a4/exampleSite/hugo.toml#L158

**Comment by colinwilson** [comment, marked as answer]
Yes, the showcase section will be independent using the `content/showcase/` directory like you've guessed. The exact layout is as yet undecided though.
